avocado/utils/disk.py

```diff
@@ -18,9 +18,26 @@
 Disk utilities
 """
 
+
 import os
+import re
+from . import process
 
 
 def freespace(path):
     fs_stats = os.statvfs(path)
     return fs_stats.f_bsize * fs_stats.f_bavail
+
+
+def get_disk_blocksize(path):
+    """Return the disk block size, in bytes"""
+    fs_stats = os.statvfs(path)
+    return fs_stats.f_bsize
+
+
+def get_disks():
+    df_result = process.system_output('lsblk --paths').decode("utf-8")
```

Comment:
Please don't hardcode `utf-8` for results. The main method `process.run()` detects the encoding so one can actually get `result.stdout_text`, which would IMO be the best option. Alternatively one can use `avocado.utils.astring.ENCODING` to at least attempt to guess the right encoding.